fragments/app-sso-client/config/client-registration.yaml

```diff
@@ -13,5 +13,6 @@ spec:
   requireUserConsent: false
   redirectURIs:
     - "http://<my-workload-hostname>/login/oauth2/code/<my-workload>"
+    - "https://<my-workload-hostname>/login/oauth2/code/<my-workload>"
   scopes:
```

Comment:
Could this be a setting?  I can definitely see a need for one or the other, not necessarily both.

Reply:
Done

Reply:
This is all looking great